netbox/extras/plugins/signals.py

```diff
@@ -3,7 +3,9 @@
 
 
 class PluginSignal(django.dispatch.Signal):
```

Comment:
If we are not using this anymore do you think it makes sense to leave it?

Reply:
I felt bad wiping it out. We'll probably need it or some variation of it, but yeah we should probably remove it given that it's not currently in use.